packages/salesforcedx-vscode-soql/package.nls.json

```diff
@@ -1,4 +1,6 @@
 {
+  "configuration_title": "Salesforce SOQL Configuration",
   "soqlCustom_soql": "SOQL Builder",
-  "soql_builder_open_new": "SFDX: Create Query in SOQL Builder"
+  "soql_builder_open_new": "SFDX: Create Query in SOQL Builder",
+  "soql_editor_remote_checks_flag": "Validate SOQL editor queries against your Org. When enabled, the SOQL editor calls the remote API to validate the query on your default Org environment and provide better syntax checks."
```

Comment:
@praksb @dobladez - do we need to say SOQL "editor" queries - or can we just say "SOQL queries"? Is this to distinguish a query built in a text editor vs. SOQL Builder?

Reply:
Good point. Yes, this flag only affects the behavior of the text editor... Should we use the full "SOQL text editor queries" ?

Reply:
Even though its applicable only to queries in the text editor, I don't think that distinction needs to be explicitly called out in the description text.

Reply:
@praksb - If I'm understanding your point, are you saying that we should just use "SOQL queries"?

Reply:
@ekapner yes. i am suggesting using just soql queries.